### ISSUE TITLE
Tooltip label not announced by screen readers for hover or focus triggers  

### ISSUE THREAD

**Issue by jcfizzler** [chakra-ui/chakra-ui#4682]
### Description

I'm using a Tooltip that is triggered by different actions (hover, click, focus). The label and aria-label are only announced by screen readers when clicking or hitting enter on the trigger element. When hovering or focusing the trigger element the screen reader only announces "clickable" (similar to issue https://github.com/chakra-ui/chakra-ui/issues/4205). I'm using NVDA  2020.2.

### Link to Reproduction

https://codesandbox.io/s/compassionate-maxwell-rn5cr?file=/src/index.tsx

### Steps to reproduce

1. Create Tooltip with different trigger actions (focus, hover, click) and a label and/or aria-label.
2. Hover or focus the trigger element.
3. The label (or aria-label) ARE NOT announced by NVDA.
4. Click or hit enter on the trigger element.
5. The label (or aria-label) ARE announced by NVDA.

### Chakra UI Version

1.6.0

### Browser

Chrome 92.0.4515.159

### Operating System

- [ ] macOS
- [X] Windows
- [ ] Linux

**Comment by anubra266** [comment]
cc @segunadebayo

**Comment by studiosciences** [comment]
The Tooltip prop documentation states:
> `aria-label`
> The accessible, human friendly label to use for screen readers. If passed, tooltip will show the content label but expose only `aria-label` to assistive technologies

That makes sense, but it doesn't work. The `aria-label` is not inserted. Confirmed it's still an issues in v2.0.0 too.

If you have a Tooltip with extra information, the current default behavior (`aria-describedBy`) is correct. However, for a case where the Tooltip is on an icon-only button, the tooltip would be providing a primary label, and therefore should be adding `aria-label` or `aria-labelledBy` instead.
